doc/api/process.markdown

```diff
@@ -137,13 +137,29 @@ Here is an example that logs every unhandled rejection to the console
         // application specific logging, throwing an error, or other logic here
     });
 
-For example, here is a rejection that will trigger the `'unhandledRejection'`
+For example, here is a rejection that will trigger the 'unhandledRejection'
```

Comment:
For consistency with the rest of these two sections. The doc is highlight inconsistent about how events are named but at least the promise sections decided on `'string'` not `` `string` `` or `` `'string'` ``